easybuild/tools/build_log.py

```diff
@@ -186,10 +186,10 @@ def exception(self, msg, *args):
 _init_easybuildlog = fancylogger.getLogger(fname=False)
 
 
-def init_logging(logfile, logtostdout=False, silent=False):
+def init_logging(logfile, logtostdout=False, silent=False, color=None):
```

Comment:
the default for `color` here should be a string value? or `None` should be handled well in `logToScreen`, maybe that's better?